app/client/components/sectionContent.tsx

```diff
@@ -65,18 +65,28 @@ const divCss = (hasNav: boolean | undefined) => css`
   }
 `;
 
+export const SelectedTopicObjectContext = React.createContext<
+  Dispatch<SetStateAction<HelpCentreNavConfig | undefined>>
+>(undefined!);
+
 export const SectionContent = (props: SectionContentProps) => {
+  const [selectedTopicObject, setSelectedTopicObject] = useState<
+    HelpCentreNavConfig | undefined
+  >(props.selectedTopicObject);
```

Comment:
`selectedTopicObject` is no longer being passed in as a prop so I think we can removed it on line 12 & initialise this state as `undefined`.